shared/common-adapters/web-view.native.js

```diff
@@ -1,8 +1,38 @@
 // @flow
+import * as React from 'react'
 import {isIOS} from '../constants/platform'
 import {WebView} from 'react-native'
 import WKWebView from 'react-native-wkwebview-reborn'
+import type {WebViewInjections, WebViewProps} from './web-view'
+import {memoize} from 'lodash-es'
 
-const wv = isIOS ? WKWebView : WebView
+const sanitize = (str?: string): string => (str ? str.replace(/\\/g, '\\\\').replace(/`/g, '\\`') : '')
 
-export default wv
+const combineJavaScriptAndCSS = (injections?: WebViewInjections) =>
+  !injections
+    ? ''
+    : `
+(function() {
+  const node = document.createElement('style')
+  document.body.appendChild(node)
+  node.innerHTML = \` ${sanitize(injections.css)} \`
```

Comment:
setting `innerHTML` gives me the heebie-jeebies. Try
```
const style = document.createElement('style');
document.head.appendChild(style);
style.type = 'text/css';
style.appendChild(document.createTextNode(css));
```

from https://stackoverflow.com/questions/524696/how-to-create-a-style-tag-with-javascript

Reply:
Sounds good! I didn't realize this option. Will do!